src/CONST.js

```diff
@@ -490,8 +490,8 @@ const CONST = {
         WIDTH: 320,
         HEIGHT: 400,
     },
-    NON_NATIVE_EMOJI_PICKER_LIST_HEIGHT: 300,
-    EMOJI_PICKER_ITEM_HEIGHT: 40,
+    NON_NATIVE_EMOJI_PICKER_LIST_HEIGHT: 295,
```

Comment:
Why does this need to change?

Reply:
Actual height of Emoji Picker is 295. So, changed the value to match value in UI.
<img width="692" alt="Screenshot 2022-11-02 at 15 04 43" src="https://user-images.githubusercontent.com/25876548/199455058-f9a03a5e-6dba-421e-9ad4-cc7da60335ec.png">

Reply:
Hmm but if 300 wasn't doing anything before, what makes us believe that 295 will do anything either?

Reply:
This value is used in the calculation of offset while scrolling with keyboard. Setting right value will help to solve [issue](https://github.com/Expensify/App/issues/8459) in scrolling.

Reply:
Got it. But ideally the value we code should actually be reflected in the UI. Why isn't it actually working?

Reply:
Good point.

Reply:
Total Height of Emoji Picker is 400. 
```
    EMOJI_PICKER_SIZE: {
        WIDTH: 320,
        HEIGHT: 400,
    },
```
Search Bar occupies 62, Bottom Skin tone Selector occupies 48. Remaining: 290